react/getting_started_with_react/rendering_techniques.md

```diff
@@ -102,12 +102,22 @@ function App() {
 ~~~
 
 <div class="lesson-note lesson-note--tip" markdown="1">
-  <h4>"Missing in props validation"</h4>
+  #### "Missing in props validation"
```

Comment:
Nit: the heading formatting is acting a little funky on the preview tool: 
<img width="749" alt="Screenshot 2024-03-05 at 17 19 28" src="https://github.com/TheOdinProject/curriculum/assets/7963776/3a516b8d-3587-43aa-b2c3-742302de6bd0">


Outdenting should fix it:

```suggestion
#### "Missing in props validation"
```